storage/innobase/xtrabackup/test/t/pxb-1961.sh

```diff
@@ -0,0 +1,65 @@
+############################################################################
+# PXB-1961: xtraback fails with encryption='n' with Percona server
+############################################################################
+
+function backup_and_restore() {
+record_db_state test
+rm -rf $topdir/backup
+mkdir -p $topdir/backup
+# Backup
+xtrabackup --datadir=$mysql_datadir --backup --target-dir=$topdir/backup 
+# Prepare
+xtrabackup --datadir=$mysql_datadir --prepare --target-dir=$topdir/backup
+stop_server
+# Restore
+rm -rf $mysql_datadir/ibdata1 $mysql_datadir/ib_logfile* \
+    $mysql_datadir/test/*.ibd
+cp -r $topdir/backup/* $mysql_datadir
+start_server
+# Verify backup
+verify_db_state test
+
+}
+if ! is_debug_server  ; then
+       skip_test "Require debug server version"
+fi
+
+. inc/common.sh
+
+MYSQLD_EXTRA_MY_CNF_OPTS="
+innodb_file_per_table
+"
+
+. inc/keyring_file.sh
+
+start_server
+
+vlog "case#1 restore table created with encryption='n'"
+mysql test <<EOF
+set global innodb_page_cleaner_disabled_debug=on;
+CREATE TABLE t1(a INT) ENGINE=INNODB ENCRYPTION='N';
+INSERT INTO t1 VALUES(100);
+EOF
+backup_and_restore
+
+vlog "case#2 check if table can be altered with encryption='n'"
+mysql test <<EOF
+ALTER TABLE t1 ENCRYPTION='Y';
+ALTER TABLE t1 ENCRYPTION='N';
+INSERT INTO t1 VALUES(100);
+EOF
+innodb_wait_for_flush_all
```

Comment:
Will this ever reach to 0 if we have `set global innodb_page_cleaner_disabled_debug=on;` ?

Reply:
once we restart server this options gets disabled.